Emby.Server.Implementations/HttpServer/WebSocketConnection.cs

```diff
@@ -218,7 +221,42 @@ private async Task ProcessInternal(PipeReader reader)
                 Connection = this
             };
 
-            await OnReceive(info).ConfigureAwait(false);
+            if (info.MessageType.Equals("KeepAlive", StringComparison.Ordinal))
+            {
+                SendKeepAliveResponse();
+            }
+            else
+            {
+                await OnReceive(info).ConfigureAwait(false);
+            }
+        }
+
+        private void SendKeepAliveResponse()
+        {
+            LastKeepAliveDate = DateTime.UtcNow;
+            SendAsync(new WebSocketMessage<string>
```

Comment:
Doesn't this need to be awaited?

Reply:
I wonder if there was a reason, but it can surely be awaited.